dune-project

```diff
@@ -1,3 +1,28 @@
-(lang dune 1.0)
-
+(lang dune 2.7)
 (name ppx_deriving)
+(version 5.2)
+
+(formatting disabled)
+(generate_opam_files true)
+(implicit_transitive_deps true) ; TODO: change to false
+
+(license MIT)
+(maintainers "whitequark <whitequark@whitequark.org>")
+(authors "whitequark <whitequark@whitequark.org>")
+(source (github ocaml-ppx/ppx_deriving))
+(documentation "https://ocaml-ppx.github.io/ppx_deriving")
+
+(package
+  (name ppx_deriving)
+  (synopsis "Type-driven code generation for OCaml")
+  (description "ppx_deriving provides common infrastructure for generating
+code based on type definitions, and a set of useful plugins
+for common tasks.
+")
+  (depends
+    (ocaml (>= 4.05))
+    (cppo :build)
+    ppx_derivers
```

Comment:
I think you can remove `ppx_derivers` safely, since it's unnecessary for the ppxlib-port: https://github.com/ocaml-ppx/ppx_derivers/issues/6

Reply:
I sent a separate PR for this then: https://github.com/ocaml-ppx/ppx_deriving/pull/244